internal/context.py

```diff
@@ -18,7 +18,7 @@ async def guild_config(self) -> dict:
             return json.loads(guild["config"])
         return {}
 
-    async def update_guild_config(self, **kwargs):
+    async def update_guild_config(self, **kwargs) -> None:
```

Comment:
Seeing how we've type-hinted *args in some other places, I think we should type hint **kwargs as well to maintain uniformity.

Reply:
They can be any type. You want me to type hint to object?

Reply:
typing.Any I guess